app/scripts/modules/core/src/pipeline/executions/execution/Execution.tsx

```diff
@@ -337,28 +344,37 @@ export class Execution extends React.Component<IExecutionProps, IExecutionState>
             { execution.isRunning && (
               <Tooltip value="Pause execution">
                 <a className="clickable" onClick={this.handlePauseClick}>
```

Comment:
I think we can make this a button instead of link and also avoid the event.stopPropagation
<button className="link">